src/main/java/com/codecool/dungeoncrawl/logic/actors/Actor.java

```diff
@@ -19,6 +23,28 @@ public void move(int dx, int dy) {
         cell = nextCell;
     }
 
+    public void getItem(Object o) {
```

Comment:
Might use full word for parameter's name .

Reply:
I think that o usually used for an object instance so it would not make the code more readable

Reply:
It's relevant ! You are right